src/row.ts

```diff
@@ -34,7 +34,7 @@ import {TabularApiSurface} from './tabular-api-surface';
 
 export interface Rule {
   column: string;
-  append: string;
+  append?: string;
```

Comment:
append shouldn't actually be required and the reason I know this is that while developing, I noticed the increment method which works properly calls createRules providing a `Rule` object without an `append` property.